src/support/map-data-services/ors-response-data-extractors/response-extractors/v2/isochrones.js

```diff
@@ -61,12 +62,20 @@ class IsochronesBuilder {
   getPolygons = () => {
     const polygons = []
     if (this.responseData.features) {
+      // When you request multiple points in an isochrone request, ors returns them all as a single array
+      // of polygon features
+      const ranges = this.responseData.metadata.query.range
+      const maxRange = Math.max(ranges)
+
       this.responseData.features.forEach((feature, index) => {
         const polygon = { geometry: feature.geometry, properties: feature.properties }
         polygon.properties.range_type = this.responseData.metadata.query.range_type
+        if (store.getters.mapSettings.alternativeIsochroneColors) {
```

Comment:
With this, color can be set both here and in `preparePolygonForView`, which is a bit confusing…
I'd propose either moving the color selection in the default case out here or moving the color selection in the alternative case to `preparePolygonForView` to have this closer together.